tests/phpunit/Presentation/TabsHtmlBuilderUnitTest.php

```diff
@@ -157,4 +190,68 @@ public function testTabsViewModelSelectsCurrentItemType(): void {
 		);
 	}
 
+	public function testSettingsViewModelContainsValuesWhenUserCanEditConfigurations(): void {
+		$config = $this->newConfig( <<<JSON
+{
+	"itemTypeProperty": "P1337",
+	"configPerItemType": {
+		"Q5976445": {
+			"facets": {
+				"P592": {
+					"type": "list"
+				}
+			}
+		}
+	}
+}
+JSON );
```

Comment:
Why are you providing the JSON? Can't we just call `newConfig()`, or am I missing a relevant detail?

Reply:
Actually, why the changes to the config code in this test class at all? Didn't the old `$config ?? new Config(),` work?

Reply:
Mainly to provide a bare minimum `Config` that does not throw an exception.

`new Config()` does not work because the tests will throw `RuntimeException: No instance of ID configured` when there are no `$config` provided. We did not encounter it before because every test cases already defined its own `$config`, so `new Config()` was never triggered.